docs/otoc/otoc_example.ipynb

```diff
@@ -0,0 +1,761 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "xmJUa-i__Vl9"
+   },
+   "source": [
+    "## Background on quantum scrambling"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "D-6CzQFH84Tr"
+   },
+   "source": [
+    "Quantum scrambling describes how interaction in a quantum system disperses local information into its many degrees of freedom. Analogous to classical chaos, scrambling manifests itself as a [butterfly effect](https://en.wikipedia.org/wiki/Butterfly_effect) wherein a local perturbation is rapidly amplified over time. The perturbation is modeled by a local operator $\\hat{O}$ (the *butterfly operator*) acting on a qubit (the *butterfly qubit*). Under a dynamical process $\\hat{U} = \\hat{U}(t)$, the butterfly operator evolves in the Heisenberg picture as\n",
+    "\n",
+    "$$\n",
+    "\\hat{O} (t) = \\hat{U}^\\dagger \\hat{O} \\hat{U}\n",
+    "$$\n",
+    "\n",
+    "Expanding the butterfly operator in a basis\n",
+    "\n",
+    "$$\n",
+    "\\hat{O} (t) = \\sum_{i = 1}^{n_p} w_i B_i \n",
+    "$$\n",
+    "\n",
+    "where $B_i = B^{(1)}_i \\otimes \\cdots \\otimes B^{(n)}_i$ are basis elements and $w_i$ are coefficients, we can describe the two different mechanisms of quantum scrambling as follows:\n",
+    "\n",
+    "- *Operator spreading* occurs when basis elements $B_i$ require a higher weight (fewer non-identity terms) on average to describe $\\hat{O}(t)$.\n",
+    "- *Operator entanglement* occurs when a larger $n_p$ (summation index) is required to describe $\\hat{O}(t)$.\n",
+    "\n",
+    "We experimentally evaluate the out-of-time order correlator (OTOC) $C(t)$ between $\\hat{O}(t)$ and a *measurement operator* $\\hat{M}$ which is a Pauli operator acting on another qubit (the *measurement qubit*)\n",
+    "\n",
+    "$$\n",
+    "C(t) := \\langle \\hat{O} ^\\dagger (t) M^\\dagger \\hat{O} (t) M \\rangle\n",
+    "$$\n",
+    "\n",
+    "over a collection of quantum circuits with microscopic differences (e.g., in the phases of gates). Operator scrambling is then reflected in the average OTOC value $\\bar{C}(t)$ over different circuits:\n",
+    "\n",
+    "- $\\bar{C}(t) = 1$ when $\\hat{O}(t)$ and $\\hat{M}$ do not overlap.\n",
+    "- $\\bar{C}(t) < 1$ when $\\hat{O}(t)$ and $\\hat{M}$ overlap.\n",
+    "- $\\bar{C}(t) \\rightarrow 0$ in the fully scrambled limit where the commutation between $\\hat{O}(t)$ and $\\hat{M}$ is completely randomized.\n",
+    "\n",
+    "Note that if operator entanglement is also present ($n_p \\gg 1$), $\\bar{C}(t)$ approaches $0$ for all circuits and their fluctuation vanishes as well.\n",
+    "\n",
+    "In what follows, we show how to build OTOC circuits, run them & compute $\\bar{C}$, then visualize results and detect operator spreading."
```

Comment:
nit why do you use "&" once but "and" in other places

Reply:
Changed & -> and